downstairs/src/region.rs

```diff
@@ -24,13 +23,66 @@ pub struct Extent {
 #[derive(Debug)]
 pub struct Inner {
     file: File,
-    meta: ExtentMeta,
+    metadb: Connection,
+}
+
+impl Inner {
+    fn flush_number(&self) -> Result<u64> {
+        let mut stmt = self
+            .metadb
+            .prepare("SELECT value FROM metadata where name='flush_number'")?;
+        let flush_number_iter = stmt.query_map([], |row| row.get(0))?;
+
+        let mut flush_number_values: Vec<u64> = vec![];
+        for flush_number_value in flush_number_iter {
+            flush_number_values.push(flush_number_value?);
+        }
+
+        assert!(flush_number_values.len() == 1);
+
+        Ok(flush_number_values[0])
+    }
+
+    fn set_new_flush_number(&self, new_flush: u64) -> Result<()> {
```

Comment:
Nit: can we call this "set_flush_number".  Plus it is shorter and I like shorter.

Reply:
Shortened in 24da54d